docs/tutorial/running-ehrql.md

```diff
@@ -58,7 +58,7 @@ For example if you type `1 + 1` and press the return key, you should see:
 
 To use ehrQL, you'll first need to import the tables that you want to interact with:
 
-    >>> from ehrql.tables.beta.core import patients, medications
+    from ehrql.tables.beta.core import patients, medications
```

Comment:
thought:

I'm guessing you've removed the `>>>` prompt to make it easier to copy-paste the code. That makes sense.

But I'm not sure if we're still best to keep the prompt indicators in, to show that this to be done in a Python interactive console.

Reply:
That's fine! Having that context as a [code block title](https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title) would be nice!

Reply:
Ah, that's actually a use of that titling I hadn't considered!

I'll look into how we might use that.